src/psyclone/tests/psyir/nodes/array_reference_test.py

```diff
@@ -441,15 +441,15 @@ def test_array_indices():
     array._children = [one.copy(), "hello"]
     with pytest.raises(InternalError) as err:
         _ = array.indices
-    assert ("ArrayReference malformed or incomplete: child 1 must by a "
-            "psyir.nodes.DataNode or Range representing an array-index "
+    assert ("ArrayReference malformed or incomplete: child 1 of 'test' must "
```

Comment:
Maybe "child 1 of array 'test'" and the same in the error message below

Reply:
Yes, that's better.